### ISSUE TITLE
Importing of flows that create Contact Fields doesn't create proper type

### ISSUE THREAD

**Issue by nicpottier** [rapidpro/rapidpro#147]
On flows that get imported that include setting Contact Fields that don't exist on an org, we don't currently export (or set on import) the type of that field, though we do create it.

For example, if you have a flow that sets a "State" field, when imported into a virgin organization, "State" is created but its type is not correctly set to "State"

**Comment by ericnewcomer** [comment]
This is something we are looking at, however a few points have been raised so far:

* We don't capture field type in the flow spec
* We should also create fields referenced in expressions in the flow, types for those are problematic

We've discussed letting expression references be created as string, but we've also leaned toward making the type for contact fields immutable. This implies a fancier way to declare the type of contact fields on import. However, what does that mean when somebody references a brand new contact field in a text box in the editor? Does that also get created? How is it's type determined?
